kotlinpoet/build.gradle.kts

```diff
@@ -37,19 +39,61 @@ kotlin {
     withJava()
   }
 
+  js {
+    browser {
+      testTask {
+        useKarma {
+          useChromeHeadless()
+        }
+      }
+    }
+    nodejs {
+      testTask {
+        useMocha()
+      }
+    }
```

Comment:
There's no need to run the tests in both the browser and on Node. Just Node is fine.